IFC4x3/Properties/d/DrillholeDiameter_0AYonBcML4YQQy0$dhvZUY/Documentation.md

```diff
@@ -0,0 +1 @@
+Diameter of the injection hole.
```

Comment:
I injection hole and drill hole interchangeable?

Reply:
Ping @JontyDunn and @hessf-ilf

Reply:
Using "drillhole" everywhere.